### DISCUSSION TITLE
Noise reduction doesn't work

### DISCUSSION THREAD

**Discussion by kjxbyz** [renovatebot/renovate#25881] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["config:base"],
  "rangeStrategy": "bump",
  "prHourlyLimit": 0,
  "ignoreDeps": ["node"],
  "packageRules": [
    {
      "allowedVersions": "18",
      "matchPackageNames": ["@types/node"]
    },
    {
      "allowedVersions": "5",
      "matchPackageNames": ["@mui/x-date-pickers"]
    },
    {
      "matchPackagePatterns": ["​tabler"],
      "groupName": "tabler"
    }
  ]
}
```

https://github.com/kjxbyz/next-admin-starter/pull/198

https://github.com/kjxbyz/next-admin-starter/pull/197

Two PRs are not merged into one PR

https://github.com/kjxbyz/next-admin-starter

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by HonkingGoose** [comment]
The links to your GitHub PRs don't work for me. Is the `kjxbyz/next-admin-starter` repository set to private?

By the way, we renamed the `config:base` preset to `config:recommended`, you should update the preset name. :wink:
